docs/src/examples/dual_quaternions.md

```diff
@@ -0,0 +1,159 @@
+# Dual quaternions
+
+## Introduction
+
+The [dual quaternions](https://en.wikipedia.org/wiki/Dual_quaternion) are an example of "biquaternions."
```

Comment:
```suggestion
The [dual quaternions](https://en.wikipedia.org/wiki/Dual_quaternion) are an example of "biquaternions".
```

Reply:
It's a silly rule, but periods are supposed to go within quotes like this.

Reply:
Oh, sorry. The quoted thing is just one word but not a sentence, so I thought that was just a typo:sweat_smile: